src/internet_identity/src/http.rs

```diff
@@ -193,6 +208,11 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
                 register_rate_limit_config.max_tokens as f64,
                 "The maximum number of `register` calls that are allowed in any time window.",
             )?;
+            w.encode_gauge(
+                "internet_identity_register_rate_limit_time_per_tokens_seconds",
+                Duration::from_nanos(register_rate_limit_config.time_per_token_ns).as_secs() as f64,
+                "Min number of nanoseconds between two register calls to not exceed the rate limit (sustained).",
```

Comment:
Given the conversion to seconds in the previous line, shouldn't it say "Min number of seconds"?

Reply:
Ah, yes. Thanks!